nixos/modules/services/web-apps/ocis.nix

```diff
@@ -0,0 +1,114 @@
+{
+  config,
+  lib,
+  pkgs,
+}:
+
+let
+  inherit (lib) types;
+  cfg = config.services.ocis;
+in
+
+{
+  options = {
+    services.ocis = {
+      enable = lib.mkEnableOption "ownCloud Infinite Scale";
+
+      package = lib.mkPackageOption pkgs "ocis-bin" { };
+
+      environmentFile = lib.mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        example = "/run/keys/ocis.env";
+        description = ''
+          An environment file as defined in {manpage}`systemd.exec(5)`.
+        '';
+      };
+
+      settings = lib.mkOption {
```

Comment:
This `settings` option is never written to a place where the user-supplied settings could actually be picked up by OCIS. It should probably be written as an environmentfile in the systemd service, no?

Reply:
yea, it was somehow always a TODO item to use it. Now it gets written as environmentFile, prepended by the other environmentFile user may want and set